lib/pages/portal/service/website_provider.dart

```diff
@@ -103,6 +104,47 @@ class WebsiteProvider with ChangeNotifier {
     }
   }
 
+  ///This method initializes the number of visits of websites with the value
+  /// accumulated until then. These data are stored in 2 lists: the list of website ids
+  /// and the list with the number of accesses, according to the index in the list
```

Comment:
```suggestion
  /// Initializes the number of visits of websites with the value stored locally.
  
  /// Because [PrefService] doesn't support storing maps, the
  /// data is stored in 2 lists: the list of website IDs ([websiteIds]) and the list
  /// with the number of visits ([websiteVisits]), where `websiteVisits[i]` is the 
  /// number of times the user accessed website with ID `websiteIds[i]`.
```